documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -4,3 +4,4 @@
 
 ### New features
 * Support for GoLang is now extended to Go 1.22.2.
+* [company_name] [solution_name] now allows to exclude dev dependencies when using the Poetry detector. See the [detect.poetry.dependency.groups.excluded](properties/detectors/poetry.md#detect.poetry.dependency.groups.excluded) property for more information.
```

Comment:
A little cleanup suggestion:
* [company_name] [solution_name] now allows exclusion of development dependencies when using the Poetry detector.

Reply:
Updated the one sentence. Would you say it's worth it to keep the second one?

Reply:
Yes indeed. I think it is always nice to have a link to the property/topic in question.